ddp-workspace/projects/ddp-pancan/src/app/components/page-with-sections/page-with-sections.component.scss

```diff
@@ -5,6 +5,7 @@
 
     &.about-us {
         .section-0 {
+            min-height: 705px;
```

Comment:
In order to the background image fits the updated section (amount of text changed).

Reply:
maybe background-size: contain; would be a better choice here: we won't depend on the text size

Reply:
fixed